src/passwordListener.java

```diff
@@ -9,9 +9,9 @@
 
 	Client_Password_Service service = new Client_Password_Service();
 	private static final String passwordValidation = 
-           "((?=.*[a-zA-Z-0-9]).{6,10})";
+           "((?=.*\\d)(?=.*[a-zA-Z]).{6,10})";
```

Comment:
This is a regular expression to validate for the password field http://regexlib.com/CheatSheet.aspx?AspxAutoDetectCookieSupport=1